server/index.js

```diff
@@ -1,11 +1,19 @@
 const express = require('express');
-const path = require('path');
+const bodyParser = require('body-parser');
+const db = require('../database/index.js');
 
 const app = express();
-
 const port = process.env.PORT || 3001;
 
-app.use(express.static(path.join(__dirname, '/../client/dist')));
+app.use(express.static(__dirname + '/../client/dist'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded());
+
+app.get('/api/photos/:offering', (req, res) => {
```

Comment:
Continuing from my comment above, this is probably where you would want to use `req.params` to get the offering id from the url. Then, you can pass the offering id into `db.retrieve` instead of hardcoding the offering id within the backend.

Reply:
To address all your comments, passing `/api/photos/4` will return an object to the page. I originally wanted the page to render dynamically with any given `/api/photos/<offering_id>` but realized I did not have a way for the user to change offering_ids on the main page. I will remove that and resume hardcoding the room output for the meantime. Thanks for the question and feedback!